docs/source/support/new-assistant.rst

```diff
@@ -33,10 +33,18 @@ We've released a new command-line tool for working with the DAML SDK: DAML Assis
 Migrating a ``da`` project to ``daml``
 ======================================
 
-To migrate an existing project to use ``daml``:
+The easiest way to migrate an existing project is to use the ``daml init`` command. To use it, simple go to the project root on the command line and run ``daml init``. This will create a ``daml.yaml`` file based on ``da.yaml``.
```

Comment:
I think this is great, but what you could do (to make it simpler) is have two headings: `Migrating using daml init` and `Migrating manually`. That lets you cut out the "the easiest way" and "alternatively", and is a little easier tos can. But I think it's totally fine without it!

Reply:
That sounds good! Will make the change